app/models/document.rb

```diff
@@ -18,6 +18,33 @@ class Document < ApplicationRecord
   # corresponds to which Document Type.
   DOCUMENT_CONTROLLERS = {
     "W-2" => Questions::W2sController,
+    "1095-A" => Questions::Form1095asController,
+    "1098" => Questions::Form1098sController,
+    "1098-E" => Questions::Form1098esController,
+    "1098-T" => Questions::Form1098tsController,
+    "1099" => Questions::Form1099sController,
+    "1099-A" => Questions::Form1099asController,
+    "1099-B" => Questions::Form1099bsController,
+    "1099-C" => Questions::Form1099csController,
+    "1099-DIV" => Questions::Form1099divsController,
+    "1099-G" => Questions::Form1099dashgsController,
+    "1099-INT" => Questions::Form1099intsController,
+    "1099-K" => Questions::Form1099ksController,
+    "1099-MISC" => Questions::Form1099miscsController,
+    "1099-R" => Questions::Form1099rsController,
+    "1099-S" => Questions::Form1099ssController,
+    "1099-SA" => Questions::Form1099sasController,
+    "1099-SSDI" => Questions::Form1099ssdisController,
+    "1099G" => Questions::Form1099gsController,
+    "5498-SA" => Questions::Form5498sasController,
+    "IraStatement" => Questions::FormIrastatementsController,
+    "property_tax_statement" => Questions::FormPropertyTaxStatementsController,
```

Comment:
the capitalization is inconsistent here. I'd be inclined to change these keys to be "IRA Statement" & "Property Tax Statement" so we have good string to use for labeling these types of documents. I'm concerned that the inconsistencies could become an issue down the road when we query documents by type.